Source/DafnyCore/AST/IndentationFormatter.cs

```diff
@@ -404,10 +302,11 @@ public class IndentationFormatter : TopDownVisitor<int>, IIndentationFormatter {
           break;
         case "reads" or "modifies" or "decreases" or "requires" or "ensures" or "invariant" or "yield": {
             if (IsFollowedByNewline(token)) {
+              // In the future, we might want to use this if reducing blockiness as well
```

Comment:
It's an internal option. Basically, instead of left-aligning expressions (which I have been doing in this formatter PR, after all, it's something only a formatter should do), reducing blockiness I allow expressions to wrap the line.
A few tests summarize the situation

"Normal" for a formatter, à la "blockly". Users don't write this style usually because without a formatter it's painful to add these spaces, and indentation might be a bit too much on small screens, but for code legibility it's can be great.
```
var x := match z {
           case 1 => 2
           case 2 => 3
         };
```
Reducing blockiness (now the default, after considering human-written code to minimize formatting disruptions and also reduce indentation) looks like this:
```
var x := match z {
  case 1 => 2
  case 2 => 3
};
```